addons/pos_adyen/controllers/main.py

```diff
@@ -19,14 +19,20 @@ def notification(self):
 
         _logger.info('notification received from adyen:\n%s', pprint.pformat(data))
         terminal_identifier = data['SaleToPOIResponse']['MessageHeader']['POIID']
-        payment_method = request.env['pos.payment.method'].sudo().search([('adyen_terminal_identifier', '=', terminal_identifier)], limit=1)
+        payment_method_sudo = request.env['pos.payment.method'].sudo().search([('adyen_terminal_identifier', '=', terminal_identifier)], limit=1)
 
-        if payment_method:
+        pos_order_id = data["SaleToPOIResponse"]["PaymentResponse"]["SaleData"]["SaleTransactionID"]["TransactionID"]
+        pos_session_id = int(pos_order_id[:5])
+        pos_config_sudo = request.env["pos.session"].sudo().browse(pos_session_id).config_id
+
+        if payment_method_sudo:
             # These are only used to see if the terminal is reachable,
             # store the most recent ID we received.
             if data['SaleToPOIResponse'].get('DiagnosisResponse'):
-                payment_method.adyen_latest_diagnosis = data['SaleToPOIResponse']['MessageHeader']['ServiceID']
+                payment_method_sudo.adyen_latest_diagnosis = data['SaleToPOIResponse']['MessageHeader']['ServiceID']
             else:
-                payment_method.adyen_latest_response = json.dumps(data)
+                payment_method_sudo.adyen_latest_response = json.dumps(data)
+                request.env["bus.bus"].sudo()._sendone(pos_config_sudo.current_session_id._get_bus_channel_name(), "ADYEN_LATEST_RESPONSE", "adyen responded")
```

Comment:
you can remove "adyen responded" if it's useless.

Reply:
based on this comment: https://github.com/odoo/odoo/pull/125593/files#r1290057003

You might need to put some payload to the bus message. To make sure the only the concerned pos session handles the notification.